app/models/organization.rb

```diff
@@ -27,6 +29,7 @@ class Organization < ApplicationRecord
   validates_attachment_content_type :favicon, content_type: %r[\Aimage\/(png|jpeg|jpg|x-icon|vnd.microsoft.icon)\z]
   validates_attachment_content_type :logo_image, :banner_image, content_type: %r[\Aimage\/(png|gif|jpeg|jpg)\z]
   accepts_nested_attributes_for :user
+  accepts_nested_attributes_for :organization_emails
```

Comment:
why this list is added?

Reply:
already fixed it in coming PRs forgot to change in this one